zerver/webhooks/travis/doc.md

```diff
@@ -1,29 +1,44 @@
+# Zulip Travis CI integration
+
 See your Travis CI build notifications in Zulip!
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
-   By default, pull request events are ignored since most people
-   don't want notifications for new pushes to pull requests.  To
-   enable notifications for pull request builds, just
-   append `&ignore_pull_requests=false` to the end of the URL.
+1. {!generate-webhook-url-basic.md!}
 
 1. Add the following to the bottom of your `.travis.yml` file:
 
     ```
     notifications:
       webhooks:
-        - <URL constructed above>
+        - <URL generated above>
     ```
 
-    Push this change to your repository. To further configure which
-    specific events should trigger a notification, see
-    [Travis CI's webhook documentation][1].
+    Push this change to your repository.
 
-[1]: https://docs.travis-ci.com/user/notifications/#Configuring-webhook-notifications
+{end_tabs}
 
 {!congrats.md!}
 
 ![](/static/images/integrations/travis/001.png)
+
+{!event-filtering-additional-feature.md!}
+
+### Configuration options
+
+By default, pull request events are ignored since most people don't want
```

Comment:
Created a configuration options section for these, but they could be organized elsewhere too.